server/resources/ProspectsResources.py

```diff
@@ -1,18 +1,24 @@
 from flask_restful import Resource, reqparse
 from models.ProspectModel import ProspectModel
+import json
 
 from flask_jwt_extended import (jwt_required, get_jwt_identity)
 
+from app import db
+
 upload_parser = reqparse.RequestParser()
-upload_parser.add_argument('prospects', type=str, location='json', required=True)
+upload_parser.add_argument('prospects', type=list, location='json', required=True)
+
+def add_owner_id(p, owner_id):
+  p['owner_id'] = owner_id
+  return p
```

Comment:
If possible, might want to add this inside ProspectModel.py for cleanliness

Reply:
I would but these aren't ProspectsModel objects, they're just dicts with data that corresponds to the fields in the model.